.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -7,14 +7,15 @@ on:
     types: [published]
 
 env:
-  CIBW_BUILD: 'cp37-* cp38-* cp39-*'
+  CIBW_BUILD: 'cp37-* cp38-* cp39-* cp310-*'
+  CIBW_SKIP: "*-musllinux*"
 
   # Python build settings
   CIBW_BEFORE_BUILD: |
-    pip install pybind11 ninja && yum install -y cmake gcc gcc-c++
+    pip install pybind11 ninja
 
   # Testing of built wheels
-  CIBW_TEST_REQUIRES: numpy==1.19.5 scipy pytest pytest-cov pytest-mock flaky
+  CIBW_TEST_REQUIRES: numpy~=1.21 scipy pytest pytest-cov pytest-mock flaky
```

Comment:
How come the NumPy version was updated?

Reply:
Version 1.19 has/had issues working with Py3.10. The previous choice for 1.19 was due to TF compatibility. With TF 2.7 now out in the wild, there is no more reason to pin to the older version (and I believe the PL repo is 1.21 in `requirements.txt` too https://github.com/PennyLaneAI/pennylane/blob/92e572f0cf8a0ba950ddd0e9440abfc1d9943c27/requirements.txt#L1).